theme/official_themes.go

```diff
@@ -0,0 +1,33 @@
+package theme
+
+import (
+	"os"
+	"path"
+
+	git "gopkg.in/src-d/go-git.v4"
+)
+
+const officialThemesRepo = "https://github.com/ArmanMazdaee/test-resumic-themes.git"
```

Comment:
Some context: gopkg.in does proxy git requests as git changed to not follow redirects aka 301/302 since 2017.
Maybe we could discover the repo by doing a request to say a URL we control, which uses the URL returned by a redirect to fetch the theme repo. Thoughts?